tests/playwright/django-ory-cloud.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect, Page } from "@playwright/test"
+import { randomEmail, randomString } from "./helpers"
+
+const login = async (page: Page) => {
+  const email = randomEmail()
+  await page.fill('[name="traits.email"]', email)
+  await page.fill('[name="password"]', randomString())
+  await page.click('[value="password"]')
+  return email
+}
+
+test.describe("django-ory-cloud", () => {
+  for (const app of [
+    { url: "http://localhost:4000/", name: "django" },
+  ]) {
+    test.describe(app.name, async () => {
+      test("able to use login and sign up", async ({ page }) => {
+        await page.goto(app.url, { waitUntil: "networkidle" })
+
+        await page.locator('text=Sign in').click()
+        await expect(page).toHaveURL(/.*\/ui\/login.*/)
+
+        await page.click('[data-testid="cta-link"]')
+        await expect(page).toHaveURL(/.*\/ui\/registration.*/)
+
+        const email = await login(page)
+          await expect(page).toHaveURL(/.*\/ui\/welcome.*/)
+          await expect(page.locator("body")).toContainText(email)
```

Comment:
What's up with the indentation here? Prettier to the rescue!

```suggestion
        const email = await login(page)
        await expect(page).toHaveURL(/.*\/ui\/welcome.*/)
        await expect(page.locator("body")).toContainText(email)
```